db/migrate/20181031105158_allow_all_suppliers_own_variant_overrides.rb

```diff
@@ -0,0 +1,12 @@
+class AllowAllSuppliersOwnVariantOverrides < ActiveRecord::Migration
+  def up
+    # This migration is fixing a detail of previous migration RevokeVariantOverrideswithoutPermissions
+    #   Here we allow all variant_overrides where hub_id is the products supplier_id
+    #   This is needed when the supplier herself uses the inventory to manage stock and not the catalog
+    owned_variant_overrides = VariantOverride.unscoped
+      .joins(variant: :product).where("spree_products.supplier_id = variant_overrides.hub_id")
+
+    owned_variant_overrides.update_all(permission_revoked_at: nil)
+  end
```

Comment:
This could have some impact on performance for users using OFN, depending on the number of `VariantOverride` in the DB. This is basically what's happening (I might be wrong, of course):

1. An `ActiveRecord::Relation` is built an assigned to `owned_variant_overrides`.
2. In line 9, this `ActiveRecord::Relation` is executed issuing an `UPDATE` statement against DB. The resulting statement will lock all variant overrides that fulfill `spree_products.supplier_id = variant_overrides.hub_id` for the duration of the update.

This means that if the number of variant overrides fulfilling that is large, users updating them at the same time we run the migration will have a bad experience. It is not until the DB transaction is finished that the lock is released.

I guess all these conditions will hardly happen together but can we check @mkllnk? We basically need to get a count and then decide.